ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -370,6 +370,13 @@
     "ReadOnly": "N",
     "AccessProfiles": "caseworker-publiclaw-judiciary"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "RoleName": "leadership-judge",
+    "ReadOnly": "N",
+    "AccessProfiles": "caseworker-publiclaw-judiciary"
+  },
```

Comment:
not sure this one should be necessary, think any `leadership-judge` should have the `judge` role (at least in publiclaw according to our config) so would be provided by a separate access profile!